deployments/mainnet/weth/configuration.json

```diff
@@ -28,7 +28,7 @@
   "assets": {
     "cbETH": {
       "address": "0xBe9895146f7AF43049ca1c1AE358B0541Ea49704",
-      "priceFeed": "0x67eF3CAF8BeB93149F48e8d20920BEC9b4320510",
+      "priceFeed": "0x86392dC19c0b719886221c78AB11eb8Cf5c52812",
```

Comment:
Note: This is not the correct price feed. I'm just using a placeholder one with 18 decimals until cbETH/ETH is deployed.